test.py

```diff
@@ -2,4 +2,9 @@
 
 if __name__ == "__main__":
     import unittest
-    unittest.main(module=None)
+    import unittest.loader
```

Comment:
can you please add comment here why the simple solution does not work? (I noticed failure on Windows, so it is probably related)

Reply:
I am not sure why it shows here as a new code since it has already been integrated to master as #510.

Reply:
I have merged master into this PR and the code you are referring to has disappeared so it has been an artifact of the master not being merged here. And to tell the truth, I no longer remember why exactly it has not worked.